Src/BootCamp.Chapter/Armor.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Armor : Item
+    {
+        readonly float  _defence;
+        public float GetDefence()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetDefence' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5138976)